### PR TITLE
Added put.md in the terms folder of `hashmap`

### PR DESCRIPTION
**Pull request by dakshdeepHERE**
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/hashmap/terms/put/put.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+It takes in two parameters:
+
+- key: the key of the key-value pair to add or update in the map.
+- value: the value of the key-value pair to add or update in the map.
+
+## Example
+
+To add a key-value pair to a `HashMap` in Kotlin, you can use the `put()` function. For example:
```
Comment (SSwiniarski) [inline content/kotlin/concepts/hashmap/terms/put/put.md:32]:
```suggestion
To add a key-value pair to a `HashMap` in Kotlin, the `put()` function can be used. For example:
```

File: content/kotlin/concepts/hashmap/terms/put/put.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```kotlin
+fun main(){
+val map = HashMap<String, Int>()
+map.put("apple", 1)
+map.put("banana", 2)
+map.put("cherry", 3)
+}
+```
+
+We can also use the index operator `[]` to add a key-value pair to the `HashMap`. For example:
```
Comment (SSwiniarski) [inline content/kotlin/concepts/hashmap/terms/put/put.md:43]:
```suggestion
The index operator `[]` can be used to add a key-value pair to a `HashMap`. For example:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE I did a first review and left a couple of comments.

**Comment by dakshdeepHERE** [comment]
done @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@dakshdeepHERE I think this is ready for a second review.

File: content/kotlin/concepts/hashmap/terms/put/put.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+It takes in two parameters:
+
+- key: the key of the key-value pair to add or update in the map.
+- value: the value of the key-value pair to add or update in the map.
+
+## Example
+
+To add a key-value pair to a `HashMap` in Kotlin, the `put()` function can be used. For example:
```
Comment (yangc95) [inline content/kotlin/concepts/hashmap/terms/put/put.md:32]:
```suggestion
To add a key-value pair to a `HashMap` in Kotlin, the `.put()` method can be used. For example:
```

File: content/kotlin/concepts/hashmap/terms/put/put.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Syntax
+
+```pseudo
+fun <K, V> MutableMap<K, V>.put(key: K, value: V): V?
+```
+
+It takes in two parameters:
+
+- key: the key of the key-value pair to add or update in the map.
+- value: the value of the key-value pair to add or update in the map.
```
Comment (yangc95) [inline content/kotlin/concepts/hashmap/terms/put/put.md:28]:
```suggestion
- The `key` of the key-value pair to add or update in the map.
- The `value` of the key-value pair to add or update in the map.
```

We can also add on here what the functionality is for the key and value. For example, the key is a unique identifier that is used to access certain values within the hash map and the value is what's actually being stored or the data that associated with the key.

**Review by yangc95 — COMMENTED** [review]
This is great and straight to the point @dakshdeepHERE 👍 I left a few comments. Once they're all resolved, please tag me (or if you have any questions)!

**Comment by dakshdeepHERE** [comment]
done @yangc95

**Review by yangc95 — APPROVED** [review]
LGTM! I made some changes here: [daf90d9](https://github.com/Codecademy/docs/pull/1521/commits/daf90d93b91fd151125d3bffb057172720fe7f98)&[4a69569](https://github.com/Codecademy/docs/pull/1521/commits/4a695696a48f0195921a2b7a019feb3c54340efa)

File: content/kotlin/concepts/hashmap/terms/put/put.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+Title: '.put()'
+Description: 'Used to add a key-value pair to a hash map.'
```
Comment (yangc95) [inline content/kotlin/concepts/hashmap/terms/put/put.md:3]:
```suggestion
Description: 'Inserts a key-value pair to a hash map.'
```

**Comment by yangc95** [comment]
🍰 @dakshdeepHERE live link: https://www.codecademy.com/resources/docs/kotlin/hashmap/put

All set 🙌 
Screen Shot 2023-01-28 at 11 04 27 PM

cc: @SSwiniarski

**Comment by dakshdeepHERE** [comment]
Thanks @yangc95 🙏🏻

**Pull request merged**